app/views/subnets/index.html.erb

```diff
@@ -24,7 +24,14 @@
         <td class="ellipsis"><%= subnet.domains.map(&:name).to_sentence %></td>
         <td><%= subnet.vlanid %></td>
         <td class="ellipsis"><%= subnet.dhcp %></td>
-        <td><%= link_to_if_authorized(hosts_count[subnet], hash_for_hosts_path(:search => "subnet.name=\"#{subnet}\"")) %>
+        <td>
+          <%=
+            # TODO: https://projects.theforeman.org/issues/36517 this only works for IPv4
```

Comment:
I changed this PR to be Refs, because I still think the real issue isn't solved. I could change this to be a unique RM issue just for the broken link, but if the count isn't correct I'm not sure anyone will click on it. The problem is that for every IPv6 subnet the count is 0. After this patch it will link to a listing that may be non-empty, but the label still says 0 so users may not click on it.